privacyidea/models.py

```diff
@@ -2474,3 +2476,204 @@ def __init__(self, username, realm, resolver, authentication,
         self.authentication = authentication
         self.first_auth = first_auth
         self.last_auth = last_auth
+
+### Periodic Tasks
+
+class PeriodicTask(MethodsMixin, db.Model):
+    """
+    This class stores tasks that should be run periodically.
+    """
+    __tablename__ = 'periodictask'
+    id = db.Column(db.Integer, Sequence("periodictask_seq"), primary_key=True)
+    name = db.Column(db.Unicode(), unique=True, nullable=False)
+    active = db.Column(db.Boolean, default=True, nullable=False)
+    interval = db.Column(db.Interval, nullable=False)
+    nodes = db.Column(db.Unicode(256), nullable=False)
+    taskmodule = db.Column(db.Unicode(255), nullable=False)
+    options = db.relationship('PeriodicTaskOption',
+                              lazy='dynamic',
+                              backref='periodictask')
+    last_runs = db.relationship('PeriodicTaskLastRun',
+                                lazy='dynamic',
+                                backref='periodictask')
+
+    def __init__(self, name, active, interval, node_list, taskmodule, options=None, id=None):
+        """
+        :param name: Unique name of the node as unicode
```

Comment:
is this the `node` where the task should run?

Reply:
You are right. The comment is wrong. It is the name of the task definition. 
The nodes, where it should run, is in the note_list.

Reply:
Right, thanks for noticing!